### ISSUE TITLE
Tacotron samplerate 22050 does not match LPCNet 16000

### ISSUE THREAD

**Issue by superhg2012** [alokprasad/LPCTron#1]
1 Do you use same training dataset to train both Tacotron and LPCNet? 

I see that you copy genarated f32 files (16000) to audio directory which is generated by T2 preprocess

And  your T2 preprocess used samplerate 22050 to generate audio, mel and linear.

Does this matter?

**Comment by alokprasad** [comment]
No Need to Train LPCNet , you can use existing model. But i guess if we train with same dataset it should work better.
Regarding sample rate i guess it should 16k , you might be right.

**Comment by superhg2012** [comment]
I am working on mandarin synthesis, so I need to train LPCNet from scratch.

**Comment by alokprasad** [comment]
Yes in that case you need to have same dataset . You need a single PCM files containing the audio samples
for training LPCNET

**Comment by superhg2012** [comment]
So, Tacotron2 still predict mel spectrogram as condition for LPCNet? In LPCNet paper, the 20 dim features is not mel spectrogram. How it works?

**Comment by lyz04551** [comment]
> So, Tacotron2 still predict mel spectrogram as condition for LPCNet? In LPCNet paper, the 20 dim features is not mel spectrogram. How it works?

For Tacotron-2+lpcnet, training lpcnet is a feature generated using raw audio or a feature generated by the tacotron-2 model

**Comment by alokprasad** [comment]
@lyz04551 feature can easily be generated from Speech , so we dont need feature generated from Tacotron2 .

**Comment by lyz04551** [comment]
> @lyz04551 feature can easily be generated from Speech , so we dont need feature generated from Tacotron2 .

So the training lpcnet model uses the characteristics of the original audio, and then uses the features obtained by Tacotron-2 to get the waveform in the inference? However, when I train other vocoders, similar to wawern, wavenet, the mel spectrum obtained by Tacotron-2 is used to train the vocoder. It seems that it is not complicated to directly use the mel spectrum of the original audio to train the vocoder. But it doesn't seem to do this.

**Comment by lmingde** [comment]
I think we should change the hop_size and n_fft if we use 16k audio since we may need to predict linear spectrum:
![linear_spec](https://user-images.githubusercontent.com/38581341/63560133-6c50c400-c587-11e9-8148-444f0238e412.png)
the left is original tacotron, right is this project's .

**Comment by lyz04551** [comment]
> I think we should change the hop_size and n_fft if we use 16k audio since we may need to predict linear spectrum:
> ![linear_spec](https://user-images.githubusercontent.com/38581341/63560133-6c50c400-c587-11e9-8148-444f0238e412.png)
> the left is original tacotron, right is this project's .

How about the audio quality of the lpctron you use?

**Comment by superhg2012** [comment]
I got bad quailty using 10000 samples for traning with LPCTron. what about you?

**Comment by lyz04551** [comment]
> I got bad quailty using 10000 samples for traning with LPCTron. what about you?

The quality of the speech I synthesized is not very good, and the background always has some harsh sounds.

**Comment by ysujiang** [comment]
> > I got bad quailty using 10000 samples for traning with LPCTron. what about you?
> 
> The quality of the speech I synthesized is not very good, and the background always has some harsh sounds.

can you give me some samples?

**Comment by a-froghyar** [comment]
Hey, could anyone share a recipe for increasing the sample rate?

**Comment by alokprasad** [comment]
@a-froghyar use sox

**Comment by a-froghyar** [comment]
Hey @alokprasad I meant regarding the training and synthesis within the repo - I'd like to train Tacotron-2 and LPCNet with 24kHz samples.

**Comment by JunenuJ** [comment]
> I got bad quailty using 10000 samples for traning with LPCTron. what about you?

sorry, I want to train tacotron2+lpcnet.And I do not understand how to train LPCNet？ Using the predict features by tacotron2 or the raw features?

**Comment by chengshaodi** [comment]
@

> 

Can you tell me the version for loctron?
